app/org/sagebionetworks/bridge/play/controllers/BaseController.java

```diff
@@ -240,10 +244,14 @@ JsonNode requestToJSON(Request request) {
             if (jsonNode != null) {
                 return mapper.convertValue(jsonNode, clazz);
             }
+            throw new InvalidEntityException("Expected JSON in the request body is missing");
```

Comment:
I think the throw is better outside the try-catch block. As is, this will throw the exception, only to have it immediately be caught and immediately be rethrown.